Neos.Neos/Resources/Private/Translations/en/Main.xlf

```diff
@@ -839,6 +839,12 @@
 			<trans-unit id="login.loggedOut.body" xml:space="preserve">
 				<source>Successfully logged out</source>
 			</trans-unit>
+            <trans-unit id="documentTree" xml:space="preserve">
+                <source>Document Tree</source>
+            </trans-unit>
+            <trans-unit id="contentTree" xml:space="preserve">
+                <source>Content Tree</source>
+            </trans-unit>
```

Comment:
Good catch! But the two lines are indented one place too far.

Reply:
Thanks for pointing out. Is fixed.